docs/quality-assurance/test-profiles.md

```diff
@@ -1,26 +1,31 @@
 # Test Engineer Profiles
 
-Our job roles adhere to the [DDAT](https://www.gov.uk/government/collections/digital-data-and-technology-profession-capability-framework#qat:-test-engineer-) QAT Test Engineer roles.
+## DDAT Job Roles
+
+Our job roles adhere to the [DDAT Test Engineer](https://www.gov.uk/guidance/test-engineer) roles.
 
 The different levels of Test Engineer are:
 
-* [Test Engineer](https://www.gov.uk/government/publications/test-engineer-skills-they-need)
-* [Senior Test Engineer](https://www.gov.uk/government/publications/senior-test-engineer-skills-they-need)
-* [Lead Test Engineer](https://www.gov.uk/government/publications/lead-test-engineer-skills-they-need)
-* [Head of Test](https://www.gov.uk/government/publications/head-of-test-skills-they-need)
+* [Test Engineer](https://www.gov.uk/guidance/test-engineer#test-engineer)
+* [Senior Test Engineer](https://www.gov.uk/guidance/test-engineer#senior-test-engineer)
+* [Lead Test Engineer](https://www.gov.uk/guidance/test-engineer#lead-test-engineer)
+
+Also the Test Manager role: 
```

Comment:
Head of Test role? As that is the job title here?

Reply:
Agree - fixed